Centralize work (#37)
bluesky/suitcase-tiff: README.md

```
pip install suitcase-tiff
```

## Quick Start

`suitcase-tiff` supports stack and series.

Using stack:

```
import suitcase.tiff_stack
docs = db[-1].documents(fill=True)
suitcase.tiff_stack.export(docs, 'my_exported_files/')
```

Using series:

```
import suitcase.tiff_series
docs = db[-1].documents(fill=True)
suitcase.tiff_series.export(docs, 'my_exported_files/')
```

> The exported file will have path `my_exported_files/8fe81a89-8103-487a-85d2-23c70af96f16-primary.tiff`. The default file prefix can be changed with the `file_prefix` keyword argument. See documentation link below.

Reviewer: Can this explain the different paths for series and stack? Also it might be helpful to clairty that it will have a path _like_ this when `8fe...` is the unique ID of the Run.

```
pip install suitcase-tiff
```

## Quick Start

`suitcase-tiff` supports stack and series methods.

> ### Using stack:

Reviewer 2: ```suggestion
### Using a stack of tiff images (multiple tiff frames in a single file):
```

```
pip install suitcase-tiff
```

## Quick Start

`suitcase-tiff` supports stack and series methods.

> Using stack:

Reviewer 2: ```suggestion
### Using stack:
```

```
import suitcase.tiff_stack
docs = db[-1].documents(fill=True)
suitcase.tiff_stack.export(docs, 'my_exported_files/')
```

> The exported file will have path:

Reviewer 2: ```suggestion
The exported file will be saved as
```

`my_exported_files/c416e70a-eb99-4964-bfc0-64e291313cce-primary-spot_img.tiff`

> Using series:

Reviewer 2: ```suggestion
### Using series:
```

```
import suitcase.tiff_series
docs = db[-1].documents(fill=True)
suitcase.tiff_series.export(docs, 'my_exported_files/')
```

> Instead of one file, the exportor create seperate file per row:

Reviewer 2: ```suggestion
Instead of one file, the exporter creates a separate file per row:
```

```
my_exported_files/c416e70a-eb99-4964-bfc0-64e291313cce-primary-spot_img-0.tiff
my_exported_files/c416e70a-eb99-4964-bfc0-64e291313cce-primary-spot_img-1.tiff
my_exported_files/c416e70a-eb99-4964-bfc0-64e291313cce-primary-spot_img-2.tiff
my_exported_files/c416e70a-eb99-4964-bfc0-64e291313cce-primary-spot_img-3.tiff
...

> 

Reviewer 2: ```suggestion
```
